### PR TITLE
Checks namespace doc as well

### PR DESCRIPTION
**Pull request by BenjaminVanRyseghem**

### CODE REVIEW

**Comment by BenjaminVanRyseghem** [comment]
It looks like this is not always working, I am currently investigating

**Comment by BenjaminVanRyseghem** [comment]
PR on hold because of this clojure issue: http://dev.clojure.org/jira/browse/CLJ-130.

A patch has been proposed so hopefully it should be fixed soon :smile:

**Comment by BenjaminVanRyseghem** [comment]
Comment addressed :smile:

File: src/bikeshed/core.clj [hunk 1]
```diff
@@ -140,14 +160,30 @@
     (let [source-files (mapcat #(-> % io/file
                                     ns-find/find-clojure-sources-in-dir)
                                (flatten (get-all project :source-paths)))
+          all-namespaces (->> source-files
+                              (map load-namespace)
+                              (filter #(not= % nil)))
```
Comment (dakrone) [inline src/bikeshed/core.clj]: You can also use a shorter version of this with `(remove nil?)` here

**Comment by dakrone** [comment]
This looks good, feel free to merge it!

**Comment by BenjaminVanRyseghem** [comment]
I will merge it as soon as the issue with meta on namespace is fixed in Clojure :smile:

**Comment by lvh** [comment]
This has now been released in 1.8.0.

**Comment by dakrone** [comment]
Merged and released 0.3.0, thanks!

**Pull request merged**